internal/git/git.go

```diff
@@ -376,10 +383,45 @@ func (g *Git) CommitAndPush(openAPIDocVersion, speakeasyVersion, doc string, act
 
 	var commitMessage string
 	if action == environment.ActionRunWorkflow {
-		commitMessage = fmt.Sprintf("ci: regenerated with OpenAPI Doc %s, Speakeasy CLI %s", openAPIDocVersion, speakeasyVersion)
 		if sourcesOnly {
 			commitMessage = fmt.Sprintf("ci: regenerated with Speakeasy CLI %s", speakeasyVersion)
+		} else {
```

Comment:
too mach nesting and branching

Nit: I prefer to avoid else, either
1. set default and then `if`
2. `if -> return`, fallback `return`